sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -48,6 +48,13 @@ def test_parallel_join_workflow(self):
     from .testdata.parallel_join import download_and_join
     self._test_pipeline_workflow(download_and_join, 'parallel_join.yaml')
 
+  def test_waston_ml_workflow(self):
+    """
+    Test compiling a Waston ML workflow.
+    """
+    from .testdata.watson_train_serve_pipeline import kfp_wml_pipeline
+    self._test_pipeline_workflow(kfp_wml_pipeline, 'watson_train_serve_pipeline.yaml')
```

Comment:
The Watson ML pipeline is a good example of a complex pipeline, but for our automated unit tests we should have a slightly less complex DSL script. The point of unit tests is to tease out individual functionalities, like volume mounts here, but the Watson ML pipeline also uses `secrets`, `image_pull_policy` and requires a GitHub Token to run, which will be the next step with the upcoming integration tests.

Reply:
the [KFP testdata](https://github.com/kubeflow/pipelines/tree/master/sdk/python/tests/compiler/testdata) folder has:
```
volume.py
volume_snapshotop_rokurl.py
volume_snapshotop_sequential.py
volumeop_basic.py
volumeop_dag.py
volumeop_parallel.py
volumeop_sequential.py
```

Reply:
Thanks, I will update the unittest.

Reply:
i updated the unit test to use `volume.py` since snapshotop and volumeop is not yet supported.